### ISSUE TITLE
Disabled step button should be focusable

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/molecules-and-light#157]
According to https://ux.stackexchange.com/questions/103239/should-disabled-elements-be-focusable-for-accessibility-purposes it is an a11y recommendation that disabled  buttons remain focusable.  @mbarlow12  would you  please make the step button focusable when it is disabled?

**Comment by jessegreenberg** [comment]
Sorry, I have definitely heard this elsewhere, but that resource might not be correct! Lets sort out https://github.com/phetsims/sun/issues/317 before looking into this issue.

**Comment by jessegreenberg** [comment]
In phetsims/sun#317 we decided that disabled buttons should stay in the navigation order,  so we can proceed with this issue.

**Comment by mbarlow12** [comment]
Handled in above commit and noted in added comments. @jessegreenberg is it worth making a new chip-away issue for common code types that can be disabled like StepButton, RoundPushButton, and RoundToggleButton?

**Comment by jessegreenberg** [comment]
Commit looks good @mbarlow12,  thanks.  I think it would be good to move that note out of ```StepButton``` to a more general design doc that collects PhET a11y considerations, Ill go ahead and move it if you agree.

**Comment by jessegreenberg** [comment]
> is it worth making a new chip-away issue for common code types that can be disabled like StepButton, RoundPushButton, and RoundToggleButton?

I don't think so @mbarlow12, this was the one of the first a11y sims that had a disabled button, and the rest of our button types are staying in the focus order by default (I checked RoundButtonView to make sure).

I think this can be closed.
